book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- As this is a book, we should have the person and date passed in the parameter as we want to able able to keep track of the person and the date at which the book was rented out and also ensure the book itself is included. so we should update the method to this below:

```ruby
 def add_rental(person, date)
    Rental.new(date, person, self)
  end
```